frontend/src/views/CensusDirectory/index.tsx

```diff
@@ -9,6 +10,28 @@ import {
 import styled from "@emotion/styled";
 import React from "react";
 
+function DetailItem(props: { label: string; children: string }) {
+  const ItemContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+  `;
+  const ItemLabel = styled.div`
+    ${fontCapsXxxs}
+    font-weight: 600;
+    font-feature-settings:
+      "clig" off,
+      "liga" off;
+    color: #959595;
```

Comment:
`color: ${colors?.gray[400]};`

Please update `gray[400]` in the Single Cell theme file from `#999999` to `#959595`.